src/test/java/io/vertx/core/eventbus/ClusterHostTest.java

```diff
@@ -14,16 +14,24 @@
 import io.vertx.core.VertxOptions;
 import io.vertx.test.core.VertxTestBase;
 import io.vertx.test.fakecluster.FakeClusterManager;
-import org.apache.commons.lang.SystemUtils;
 import org.junit.Assume;
 import org.junit.Test;
 
 public class ClusterHostTest extends VertxTestBase {
+  private static final String OS_NAME = getOsName();
 
   @Override
   public void setUp() throws Exception {
     super.setUp();
-    Assume.assumeTrue(SystemUtils.IS_OS_LINUX);
+    Assume.assumeTrue(OS_NAME.startsWith("Linux") || OS_NAME.startsWith("LINUX"));
+  }
+
```

Comment:
can you instead use Netty's `PlatformDependant` util class ?

Reply:
Yes, done.